docs/docs/casper/overview.md

```diff
@@ -57,6 +59,14 @@ The "Logic" folder hosts a suite of mathematical calculations which are primaril
 
 An "Entrypoint" file consolidates all contract entrypoints within a single trait, streamlining the organization of key functionalities. This modular approach enhances code clarity and accessibility, providing a centralized location for developers to locate and understand the various entrypoints available within the contract.
 
+#### Events
+
+In the "Events" section, we define types of notifications that our contract sends out to external systems. These notifications, or events, help external parties keep track of what's happening in our protocol.
+
+#### Errors
+
+In the "Errors" section, we define different types of errors that might occur while our contract is running. Instead of just crashing, the contract reports these issues using clear error codes.
```

Comment:
A more formal version for the context of blockchain could be
crashing -> reverting transactions on panic